Documentation/policy/language.rst

```diff
@@ -566,8 +579,25 @@ Topic
 
   If omitted or empty, all topics are allowed.
 
-Only allow producing to topic empire-announce
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Only allow producing to topic empire-announce using Role
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
This needs a sentence here that describes the motivation of role and when to use roles and when to use apiKeys.

Reply:
Planning to add this:
```PortRuleKafka is a list of Kafka protocol constraints. All fields are optional,
if all fields are empty or missing, the rule will match all Kafka messages.
There are two ways to specify the Kafka rules. We can choose to specify a
high level "produce" or "consume" role to  a topic or choose to specify a more
low leve kafka protocol specific apiKeys. Thus the rule allows us to specify both
high level role constructs as well as low level Kafka protocol constructs.```

Reply:
done, please let me know if you think this needs to be improved.